src/atoms/google-analytics/google-analytics.tsx

```diff
@@ -25,50 +22,205 @@ const DefaultGoogleAnalytics = ({ config }) => {
   return <Analytics type="googleanalytics" targets={gaConfig} />;
 };
 
-const GoogleAnalytics4 = ({ config }) => {
-  const Ga4ConfigObject = get(config, ["opts", "featureConfig", "ga4Config"], null);
-  return <Analytics type="googleanalytics" targets={Ga4ConfigObject} config="./ga4.json" data-credentials="include" />;
+const GoogleAnalytics4 = ({ gaId }) => {
+  const getGa4Config = (gaId) => {
+    // https://www.thyngster.com/how-to-track-amp-pages-with-google-analytics-4
+    return {
+      vars: {
+        GA4_MEASUREMENT_ID: gaId,
+        GA4_ENDPOINT_HOSTNAME: "www.google-analytics.com",
+        DEFAULT_PAGEVIEW_ENABLED: true,
+        GOOGLE_CONSENT_ENABLED: false,
+        WEBVITALS_TRACKING: false,
+        PERFORMANCE_TIMING_TRACKING: false,
+        SEND_DOUBLECLICK_BEACON: false,
+        documentLocation:
```

Comment:
https://www.thyngster.com/how-to-track-amp-pages-with-google-analytics-4

this is the config returned by the endpoint https://amp.analytics-debugger.com/ga4.json